tests/functional/centos/7/cluster/hosts

```diff
@@ -17,3 +17,6 @@ ceph-client0
 
 [mgrs]
 ceph-mgr0
+
+[nfss]
+nfs0
```

Comment:
@leseb At the moment, this is fine as it is, but I was wondering why shouldn't this be ceph-nfs0? I noticed here (https://github.com/ceph/ceph-ansible/commit/298a63c437b2cf477655706e46e288a8ac743017#diff-23b6f443c01ea2efcb4f36eedfea9089R297) the "ceph" was left on. When I removed that ceph, and changed the host to ceph-nfs0, i got the error that host ceph-nfs0 couldn't be found.